docs(webhosting): update docs webhosting product (#1477)
scaleway/docs-content: managed-services/webhosting/concepts.mdx

## FTP account

> FTP, or File Transfer Protocol, is a protocol that allows you to transfer files between different computers over the Internet. It allows you, as part of your Web Hosting plan, to transfer your files from your computer to scaleway.com - e.g. to update your website. It also works the other way around: you can recover the files stored on Scaleway.com and transfer them to your personal computer, e.g. to make a backup.

Reviewer: ```suggestion
FTP, or File Transfer Protocol, is a protocol that allows you to transfer files between different computers over the Internet. It allows you, as part of your Web Hosting plan, to transfer your files from your computer to Scaleway.com - e.g. to update your website. It also works the other way around: you can recover the files stored on Scaleway.com and transfer them to your personal computer, e.g. to make a backup.
```

> FTP, or File Transfer Protocol, is a protocol that allows you to transfer files between different computers over the Internet. It allows you, as part of your Web Hosting plan, to transfer your files from your computer to scaleway.com - for example, to update your website. It also works the other way around: you can recover the files stored on Sscaleway.com and transfer them to your personal computer, for example to make a backup.

Reviewer: ```suggestion
FTP, or File Transfer Protocol, is a protocol that allows you to transfer files between different computers over the Internet. It allows you, as part of your Web Hosting plan, to transfer your files from your computer to scaleway.com - for example, to update your website. It also works the other way around: you can recover the files stored on Scaleway.com and transfer them to your personal computer, for example to make a backup.
```

## php.ini

> The `php.ini` file is the configuration file used for applications that require PHP. It contains variable configurations such as upload size, file timeout, and resource limits. For Scaleway Web Hosting, we provide PHP with the most common values preconfigured. If required, you can [customize the configuration of PHP](/webhosting/classic/how-to/configure-php/) towards your needs.

Reviewer: ```suggestion
The `php.ini` file is the configuration file used for applications that require PHP. It contains variable configurations such as upload size, file timeout, and resource limits. For Scaleway Web Hosting, we provide PHP with the most common values preconfigured. If required, you can [customize the configuration of PHP](/webhosting/classic/how-to/configure-php/) to suit your needs.
```

> The `php.ini` file is the configuration file used for applications that require PHP. It contains the configuration of variables such as upload sizes, file timeouts, and resource limits. For Scaleway Web Hosting, we provide PHP with the most common values preconfigured. If required, you can [customize the configuration of PHP](/webhosting/classic/how-to/configure-php/) towards your needs.

Reviewer: ```suggestion
The `php.ini` file is the configuration file used for applications that require PHP. It contains variable configurations such as upload size, file timeout, and resource limits. For Scaleway Web Hosting, we provide PHP with the most common values preconfigured. If required, you can [customize the configuration of PHP](/webhosting/classic/how-to/configure-php/) towards your needs.
```
